packages/experimental-app-router/src/server/routeHandler/index.ts

```diff
@@ -1,12 +1,13 @@
 import { notFound } from 'next/navigation.js';
+import * as server from 'next/server.js';
 import { tokenHandler } from './tokenHandler.js';
 
 export async function GetFn(req: Request) {
   const { pathname } = new URL(req.url);
 
   switch (pathname) {
     case '/api/faust/token': {
-      return tokenHandler(req);
+      return tokenHandler(req, server);
```

Comment:
Wierd issue with module resolution. Unless we dont pass the import here it will fail to build! 🫨